herokux/provider.go

```diff
@@ -250,6 +250,7 @@ func New() *schema.Provider {
 			//"herokux_connect": dataSourceHerokuxConnect(),
 			"herokux_postgres_mtls_certificate": dataSourceHerokuxPostgresMTLSCertificate(),
 			"herokux_registry_image":            dataSourceHerokuxRegistryImage(),
+			"herokux_addons":                    dataSourceHerokuxAddons(),
```

Comment:
Please move this up above `herokux_postgres_mtls_certificate` so the list is in alphabetical order.

Reply:
Updated

Reply:
Thank you!